app-backup/casync/casync-2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="casync - Content-Addressable Data Synchronization Tool - is a tool similar to rsync for backing up and updating directory trees"
+HOMEPAGE="https://github.com/systemd/casync"
+SRC_URI="https://github.com/systemd/casync/archive/v${PV}.tar.gz"
+
+inherit meson eutils
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="amd64"
+IUSE="fuse selinux doc"
+
+DEPEND="app-arch/zstd
```

Comment:
We don't seem to be carrying this.

Reply:
oops, I have a portage-stable pr I forgot to make.